SECURITY.md

```diff
@@ -0,0 +1,64 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+We take the security of Boston Open Source's website and community infrastructure seriously. If you discover a security vulnerability, please follow these steps:
+
+1. **DO NOT** create a public GitHub issue
+2. Report via one of these secure channels (in order of preference):
+   - Send a private message on [Zulip](https://osdc.zulipchat.com/#narrow/stream/406743-boston) to the maintainers
+   - Use GitHub's private security advisory reporting
+   - Direct message on [Mastodon](https://floss.social/@bostonopen)
```

Comment:
This will only go to @orblivion but that's ok with me. Ideally a second person would get these messages somehow. It doesn't have to be me. 😄

Reply:
"Between you, me, and the floss.social server mods :sweat_smile:" What do you think of that @iAnonymous3000 ? Same goes for Zulip though maybe that's a higher standard of trust.

Reply:
Sure, I’m happy to handle all the security-related tasks! You can tag me directly instead of saying "all maintainers" if that makes things easier. Also, if anyone needs to reach out privately, they can use my [Signal](https://signal.me/#eu/tao35l1vQxmrWcie6iW3dXlcFo0t3Pzae7WFHEMJnAIqKn2WxP5KnKKq5ClE-ZDD)  - this way the reports reach only to me (easy & secure).

Let me know what you think!

Reply:
My specific point wrt Mastodon is not about convenience. It's that if your concern is about private disclosure, I want to make sure you feel that it's sufficiently private.

Reply:
I don’t think Mastodon is sufficiently private for vulnerability disclosures. It would be better to stick with GitHub’s Security Advisories or Signal.

Happy to update the policy accordingly if everyone agrees!